CONTRIBUTING.md

```diff
@@ -1,16 +1,17 @@
-# Contributing guide
-Follow one of the instructions here to help us develop protostar! 
-Before submitting an issue/pull request, please make sure to check out those:
-   - [issues](https://github.com/software-mansion/protostar/issues) 
-   - [project cards](https://github.com/software-mansion/protostar/projects)
-   - [pull requests](https://github.com/software-mansion/protostar/pulls)
-   
-to make sure that the feature/bug has not already been addressed, or is being worked upon.
+# Contributing Guide
+## Documentation fixes
+If you spot a typo, broken link, a mistake in the documentation, feel free to [create a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+## New features or fixes
+Check if your issue is listed on the [Issues](https://github.com/software-mansion/protostar/issues) page. If the issue exists and is assigned to somebody, it will be resolved in the near future. Otherwise:
+1. Create a new issue.
+2. Read [the development guide](https://github.com/software-mansion/protostar#development)
+3. Use `master` as a base branch for fixes and `development` as a base branch for features.
+4. Update the documentation if necessary.
+5. Test your PR locally by running `poe ci`.
+6. [Create a pull request from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
 
-Our development guide can be found [here](https://starknetpy.readthedocs.io/en/latest/development.html)
-
-## If you want to contribute a feature, fix a bug, or suggest solution:
-- Follow our pull request/issue template to make it easier for us to verify your contributions 
-- Make sure to document and test the changes
-- Make a pull request with `development` as base branch
- 
\ No newline at end of file
+## Generic contribution
+If you want to help develop this project, and you don't know where to start, consider:
+1. checking issues with the label [`good first issue`](https://github.com/software-mansion/protostar/labels/good%20first%20issue)
+1. increasing test coverage
+1. fixing small bugs or adding small features
```

Comment:
Change it to `-`, You could also add info about connecting with us on the starkware discord